libs/server-sdk/tests/rule_tests.cpp

```diff
@@ -67,7 +67,7 @@ TEST_P(AllOperatorsTest, Matches) {
                                                                *store, stack);
     ASSERT_EQ(result, param.expected)
         << context.Get("user", "attr") << " " << clause.op << " "
-        << clause.values << " should be " << param.expected;
+        << Value(clause.values) << " should be " << param.expected;
 }
```

Comment:
ADL ambiguity error for `operator<<` thrown by MSVC. Hoping this resolves the ambiguity.